lib/transformVars.js

```diff
@@ -36,8 +36,15 @@ function getSassValue(sassVar, options) {
 
     case 'SassColor': {
       const {
-        r, g, b, a,
+        r, g, b, a, hex
       } = value;
+
+      // return the hex value if specified.
+      if(options.hex){
+        return hex;
```

Comment:
Not so sure about this change since it will force all colors to be returned as a hex, even colors with an alpha value. I would think we'd still want to return `rgba` in those cases.